src/module/loaddata.js

```diff
@@ -0,0 +1,35 @@
+import { taskStorage } from './newdata.js';
+
+const bkList = document.querySelector('.listTodo');
+
+let displayList;
+export default displayList = () => {
+  bkList.innerHTML = '';
+  for (let i = 0; i < taskStorage.length; i += 1) {
+    bkList.innerHTML += `
+    <div class='align-row'>
+      <div class='label-checkbox'>
+        <input type="checkbox" id="check" name="check" >
+        <label class="label" id=${taskStorage[i].keys} for="check">${taskStorage[i].description}</label>
+      </div>
+     <div>
+        <span class="delete" id=${i}><i class="fa fa-trash" aria-hidden="true"></i></span>
+        <span class="edit"><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span>
+     </div>
+    </div>
+    <hr>
+  `;
+  }
+
+  const deleteToDo = () => {
+    const remoVingBtnsEl = [...document.getElementsByClassName('delete')];
+    remoVingBtnsEl.forEach((item) => {
+      item.addEventListener('click', (e) => {
+        taskStorage.splice(e.target.id, 1);
```

Comment:
- [ ] The delete method only works for the first item on the list.  For example, when I delete the third item, it still deletes the first item. Also, doesn't update the indexes of other items.  Kindly fix it.
![image](https://user-images.githubusercontent.com/91486791/236198404-78f07503-786a-48f6-a1aa-7546e238b684.png)
![image](https://user-images.githubusercontent.com/91486791/236198603-9f6983b4-dfc3-49df-a4b7-e6e47f4cfeb9.png)